src/models/UserMapper.php

```diff
@@ -488,5 +488,116 @@ public function editUser($user, $userId) {
         return false;
     }
 
+    /**
+     * Function to get just the user ID
+     *
+     * @param string $username The username of the user we're looking for
+     * @return $user_id The user's ID (or false, if we didn't find her)
+     */
+    public function getUserIdFromUsername($username) {
+        $sql = "select ID from user where username = :username";
```

Comment:
`getUserByUsername()` also checks for active=1. This method should do the same I think.